### DISCUSSION TITLE
What do you use to format slint files?

### DISCUSSION THREAD

**Discussion by j-piccolo** [slint-ui/slint#1527] [Q&A] [answered]
title, normally the vscode language extensions i use include a formatter by default, slint doesn't seem to have one.

**Comment by ogoffart** [comment, marked as answer]
There is some work in progress of a formatter in the repository:  https://github.com/slint-ui/slint/tree/master/tools/fmt

**Reply by j-piccolo** [reply]
Nice, thanks.
